ntoskrnl/kdbg/kdb_cli.c

```diff
@@ -3622,7 +3623,8 @@ KdbpCliInit(VOID)
     }
 
     /* Load file into memory */
-    Status = ZwReadFile(hFile, 0, 0, 0, &Iosb, FileBuffer, FileSize, 0, 0);
+    ByteOffset.QuadPart = 0;
```

Comment:
These do not mean the same thing. You're not being more explicit, you're performing a different operation.

Reply:
Then, you accept s/0/NULL/g, but debate ByteOffset use in your CORE-13910 comment, or do you refuse both?

Reply:
0->NULL is fine